setup.py

```diff
@@ -14,7 +14,7 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "redis>=3.2.1",
+        "redis>=3.2.1,!=4.5.2",  # https://github.com/redis/redis-py/issues/2629
```

Comment:
@mcayanan should we use `!=4.5.2` or `<4.5.2`

Reply:
I don't know. Seems like they intend to fix it on the redis side. I guess this implies that we'd grab a later version of 4.5.2 if they end up resolving.

Reply:
lets stick with `!=` with the hopes they fix it before the next release